lib/repl.js

```diff
@@ -794,7 +794,16 @@ function complete(line, callback) {
     filter = match[1];
     var dir, files, f, name, base, ext, abs, subfiles, s;
     group = [];
-    var paths = module.paths.concat(require('module').globalPaths);
+    let paths = [];
+    if (completeOn === '.') {
+      group = ['./', '../'];
```

Comment:
Could this include all directories in the current working directory? Bash does this for example with executables.

Also this should work with files starting with `.`, right?